test/e2e/servicebindingrequest_test.go

```diff
@@ -439,12 +470,14 @@ func serviceBindingRequestTest(
 	steps []Step,
 ) {
 	// making sure resource names employed during test are unique
+	name := "e2e-service-binding-request"
 	randomSuffix := rand.Int()
 	csvName := fmt.Sprintf("cluster-service-version-%d", randomSuffix)
 	sbrName := fmt.Sprintf("e2e-service-binding-request-%d", randomSuffix)
 	resourceRef := fmt.Sprintf("e2e-db-testing-%d", randomSuffix)
 	secretName := fmt.Sprintf("e2e-db-credentials-%d", randomSuffix)
 	appName := fmt.Sprintf("e2e-application-%d", randomSuffix)
+	serviceBindingRequestNamespacedName := types.NamespacedName{Namespace: ns, Name: name}
```

Comment:
Consider naming using `sbr`, that's a well known abbreviation. :-)

Reply:
Outdated